content/id/docs/tasks/administer-cluster/manage-resources/memory-constraint-namespace.md

```diff
@@ -0,0 +1,272 @@
+[LimitRange](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#limitrange-v1-core). Jika sebuah Pod tidak memenuhi batasan yang ditentukan oleh LimitRange,
+Pod tersebut tidak dapat dibuat pada Namespace tersebut.
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+Tiap Node dalam klastermu harus memiliki setidaknya 1 GiB memori.
+
+
+
+
+<!-- steps -->
+
+## Membuat sebuah Namespace
+
+Buat sebuah Namespace sehingga sumber daya yang kamu buat pada latihan ini
+terisolasi dari komponen lain pada klatermu.
+
+```shell
+kubectl create namespace constraints-mem-example
+```
+
+## Membuat LimitRange dan Pod
+
+Berikut berkas konfigurasi untuk sebuah LimitRange:
+
+{{< codenew file="admin/resource/memory-constraints.yaml" >}}
+
+Membuat LimitRange:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints.yaml --namespace=constraints-mem-example
+```
+
+Melihat informasi mendetail mengenai LimitRange:
+
+```shell
+kubectl get limitrange mem-min-max-demo-lr --namespace=constraints-mem-example --output=yaml
+```
+
+Keluaran yang dihasilkan menunjukkan batasan minimum dan maksimum dari memori seperti yang diharapkan. Tetapi
+perhatikan hal berikut, meskipun kamu tidak menentukan nilai semula pada berkas konfirugasi untuk 
+LimitRange, nilai tersebut akan dibuat secara otomaris.
+
+```
+  limits:
+  - default:
+      memory: 1Gi
+    defaultRequest:
+      memory: 1Gi
+    max:
+      memory: 1Gi
+    min:
+      memory: 500Mi
+    type: Container
+```
+
+Mulai sekarang setiap Kontainer yang dibuat pada Namespace constraints-mem-example, Kubernetes
+akan menjalankan langkah-langkah berikut:
+
+* Jika Kontainer tersebut tidak menentukan permintaan dan limit memori, pasangkan nilai permintaan 
+dan limit memori pada Kontainer.
+
+* Memastikan Kontainer memiliki permintaan memori yang lebih besar atau sama dengan 500 MiB.
+
+* Memastikan Kontainer memiliki limit memori yang lebih kecil atau kurang dari 1 GiB.
+
+Berikut berkas konfigurasi Pod yang memiliki satu Kontainer. Manifes Kontainer
+menentukan permintaan memori 600 MiB dan limit memori 800 MiB. Nilai tersebut memenuhi
+batasan minimum dan maksimum memori yang ditentukan oleh LimitRange.
+
+{{< codenew file="admin/resource/memory-constraints-pod.yaml" >}}
+
+Membuat Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/admin/resource/memory-constraints-pod.yaml --namespace=constraints-mem-example
+```
+
+Memastikan Kontainer pada Pod sudah berjalan:
+
+```shell
+kubectl get pod constraints-mem-demo --namespace=constraints-mem-example
+```
+
+Melihat informasi mendetail tentang Pod:
+
+```shell
+kubectl get pod constraints-mem-demo --output=yaml --namespace=constraints-mem-example
+```
+
+Keluaran yang dihasilkan menunjukkan Kontainer memiliki permintaan memori 600 MiB dan limit memori
```

Comment:
Keep the API name "Container"
```suggestion
Keluaran yang dihasilkan menunjukkan Container memiliki permintaan memori 600 MiB dan limit memori
```

Reply:
All resolved in e323b8c5920d6ae02c050306495e784b8e19ad9f